### ISSUE TITLE
Pip Drain Rate Fluctuates After Maxed Out In Desert

### ISSUE THREAD

**Issue by DougLefelhocz** [jasonrohrer/OneLifeData7#892]
[Video](https://clips.twitch.tv/TangibleToughWalrusTF2John)

There does exist one point where he walks on a badlands tile, but after that 6.0 seconds per food pip gets observed, with it dropping to 5.6 seconds per food pip also observed.  I didn't see the temperature move, so it could be Hetuw mod not having accurate calculations.  But, just in case it's not, I'm posting this.

**Comment by DougLefelhocz** [comment]
I went on a server that was empty with the default client. I walked into a desert. After my temperature maxed out, I had 4.0 seconds per food pip. I walked around some more and got numbers like 4.3 and 4.7 seconds per food pip. So, this is something happening as a result of the default code, and not some mod. I don't know if something similar happens in other biomes or not.

**Comment by DougLefelhocz** [comment]
I walked around checking and stopping in jungles, badlands, prairies, grasslands, and swamps, and each time got 6.9 seconds per food pip, which sounds right (unless 6.9 isn't the exact number expected).  I checked out tundra and got 4.8 seconds per food pip, again moving from spot to spot.  So, this seems a problem with how deserts work only.  

I briefly stood in a desert and didn't move, and the second per food pip number seemed constant if I checked and checked it again without moving.  I did also make sure to step only on desert tiles when the fluctuating numbers got observed.
